builder/src/main/java/cz/xtf/builder/db/AbstractDatabase.java

```diff
@@ -26,12 +32,49 @@ public abstract class AbstractDatabase extends DefaultStatefulAuxiliary {
     protected boolean withReadinessProbe;
     protected boolean withStartupProbe;
 
+    private Supplier<String> deploymentConfigName;
```

Comment:
What is advantage of `Supplier<String>` vs. `String` in this case? It is not consistent with other usage. Does it bring some added value?

Reply:
Honestly I don't remember why I did that .... tnks for spotting it .... reverted

Reply:
thanks